repository/process.go

```diff
@@ -18,6 +34,17 @@ func ProcessRepos(gitxargsConfig *config.GitXargsConfig, repos []*github.Reposit
 
 	for _, repo := range repos {
 		wg.Add()
+
+		go func() {
+			for {
+				select {
+				case pr := <-gitxargsConfig.PRChan:
```

Comment:
NIT: should you also listen on a "done" message to shut down this goroutine after everything has been processed?